### DISCUSSION TITLE
Is there a way to just display image using path?

### DISCUSSION THREAD

**Discussion by tarunjain1st** [community/community#78094] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how to add images in readme using absolute/relative path after storing image to GitHub repo.
* able to use drag option which upload image to githubuserbase, but it can't be delete

**Comment by Gecko51** [comment]
Yeah, for images you actually want to keep under your own control, commit them into the repo and reference them with a relative path. Drag-drop in the editor uploads to `user-attachments` on GitHub's CDN, which is separate from your repo and there's no UI to delete those.

Say you drop `diagram.png` into `docs/images/` in your repo. In your `README.md` you can use any of these:

```markdown
![App architecture](docs/images/diagram.png)
![App architecture](./docs/images/diagram.png)
![App architecture](/docs/images/diagram.png)
```

All three work when GitHub renders the README on the repo page. Relative is usually cleanest because it keeps working if you fork or move the repo.

If you need the image to work outside GitHub's rendered view (for example in a Jekyll-built GitHub Pages site, or an external doc), use the raw URL instead:

```markdown
![App architecture](https://raw.githubusercontent.com/USER/REPO/main/docs/images/diagram.png)
```

A couple of gotchas worth knowing:

- HTML `<img>` works too if you need width/height or alignment: `<img src="docs/images/diagram.png" width="400">`.
- Case sensitivity matters. `Diagram.PNG` and `diagram.png` are different files on the GitHub side, even if Windows/macOS happily treats them as the same locally.
- If the image is in a private repo, the raw URL won't load from outside without a token. Stick to the relative path in that case.

Once committed, deleting the image is just `git rm docs/images/diagram.png` and a push. Full control, unlike the drag-drop uploads.
